### PR TITLE
Feature/filter tracking

### PR DESCRIPTION
**Pull request by JamesWChan**
Hi @punalsc,

Think this is all good to merge, sorry about the mix up!

Thanks!

### CODE REVIEW

File: test/filterTracking.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import {buildFilterTrackingObj} from '../src/modules/filterTracking/buildFilterTrackingObj';
+import {cleanLabelName} from '../src/modules/filterTracking/cleanLabelName';
+import {checkFilters} from '../src/modules/filterTracking/checkFilters';
+
+document.body.innerHTML =
+    '<li id="subjects">'+
```
Comment (punalsc) [inline test/filterTracking.test.js]: Most of the DOM is not being used in the test. So my advice would be to just use the nodes that are required for the test.

File: test/filterTracking.test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import {buildFilterTrackingObj} from '../src/modules/filterTracking/buildFilterTrackingObj';
```
Comment (punalsc) [inline test/filterTracking.test.js]: Would suggest, adding default values if no parameters are set.

File: test/filterTracking.test.js (hunk 1 above)
Comment (JamesWChan) [reply test/filterTracking.test.js]: Done

File: test/filterTracking.test.js (hunk 2 above)
Comment (JamesWChan) [reply test/filterTracking.test.js]: Not required

**Pull request merged**